docs/en/enterprise-edition/rule-reference/iac-security/public-exposure/appsec-gcp-100.adoc

```diff
@@ -59,7 +59,7 @@ To retrieve the existing policy and copy it to a local file:
 Replace * PROJECT-ID* with the project ID where the BigQuery table lives.
 Replace * DATASET* with the name of the BigQuery dataset that contains the table.
 Replace * TABLE* with the table name.
-Next, locate and remove the IAM bindings with either * allUsers* or * allAuthenticatedUsers* depending on your Checkov error.
+Next, locate and remove the IAM bindings with either * allUsers* or * allAuthenticatedUsers* depending on your Cortex Cloud error.
```

Comment:
I assume it removed as it mentioned in fix-runtime and removed in a different PR

Reply:
Yes, that's right. This was removed in a subsequent PR. Marking this as fixed/resolved